lang/en/typeshed/stdlib/builtins.pyi

```diff
@@ -141,7 +141,14 @@ class type(object):
         self, name: str, bases: Tuple[type, ...], dict: dict[str, Any], **kwds: Any
     ) -> None: ...
     @overload
-    def __new__(cls, o: object) -> type: ...
+    def __new__(cls, o: object) -> type:
```

Comment:
Type class needs a top leve docstring, else we don't show it in the API documentation.

Reply:
Placeholder added here [dfc60f2](https://github.com/microbit-foundation/micropython-microbit-stubs/pull/101/commits/dfc60f2f8665e8c8d24ab850545038f4c3d69662).